microconfig/utils/src/test/java/io/microconfig/utils/StringUtilsTests2.java

```diff
@@ -0,0 +1,66 @@
+package io.microconfig.utils;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.Assertions;
+import java.util.ArrayList;
+
+public class StringUtilsTests2 {
```

Comment:
If they already have a set of existing tests for the class then you will have to merge these in. Look at their existing test it seems that they useJUnit5 so you will also have to refactor these to use the new library. There is an enhancement ticket about Junit5 which you will be able to link it to.

Reply:
Copied their tests over to this class and changed the style